### DISCUSSION TITLE
`-off` variant for more icons

### DISCUSSION THREAD

**Discussion by jguddas** [lucide-icons/lucide#3810] [Ideas]
I'm happy to submit a PR adding `-off` versions for icons.
Is this something you are interested in seeing?

It would be awesome to know which icons should and shouldn't get an `-off` version and document the reasons in the icon guide.

![image](https://user-images.githubusercontent.com/25524993/214582183-ae33e6b7-361a-4181-bb4a-ac588ae307dc.png)

#### Currently supported are those 44/824 icons:

![alarm-clock-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/alarm-clock-off.svg) ![bean-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/bean-off.svg) ![bell-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/bell-off.svg) ![bluetooth-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/bluetooth-off.svg) ![calendar-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/calendar-off.svg) ![camera-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/camera-off.svg) ![candy-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/candy-off.svg) ![cigarette-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/cigarette-off.svg) ![cloud-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/cloud-off.svg) ![dna-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/dna-off.svg) ![ear-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/ear-off.svg) ![egg-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/egg-off.svg) ![eye-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/eye-off.svg) ![flag-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/flag-off.svg) ![flashlight-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/flashlight-off.svg) ![flask-conical-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/flask-conical-off.svg) ![heart-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/heart-off.svg) ![hop-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/hop-off.svg) ![image-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/image-off.svg) ![lightbulb-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/lightbulb-off.svg) ![link-2-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/link-2-off.svg) ![locate-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/locate-off.svg) ![map-pin-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/map-pin-off.svg) ![megaphone-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/megaphone-off.svg) ![mic-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/mic-off.svg) ![milk-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/milk-off.svg) ![monitor-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/monitor-off.svg) ![navigation-2-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/navigation-2-off.svg) ![navigation-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/navigation-off.svg) ![nut-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/nut-off.svg) ![phone-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/phone-off.svg) ![pin-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/pin-off.svg) ![power-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/power-off.svg) ![screen-share-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/screen-share-off.svg) ![server-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/server-off.svg) ![shield-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/shield-off.svg) ![star-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/star-off.svg) ![timer-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/timer-off.svg) ![vibrate-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/vibrate-off.svg) ![video-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/video-off.svg) ![wheat-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/wheat-off.svg) ![wifi-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/wifi-off.svg) ![wine-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/wine-off.svg) ![zap-off](https://raw.githubusercontent.com/lucide-icons/lucide/main/icons/zap-off.svg)

**Comment by karsa-mistmere** [comment]
I think `-off` icons should be added on a case by case basis, we can't really make a general rule whether an icon needs an off variant, that is: the general rule is, an off icon can be added if we can find a legit use case for it.

E.g.:
- for sure, `webcam-off` would be nice to have for the same reasons we have a `mic-off` and a `video-off` icon.
- Further airplay icons I'm not so sure about, especially since the [design itself is protected](https://developer.apple.com/design/human-interface-guidelines/technologies/airplay/icons/), so we shouldn't have added it in the first place, I'm afraid.
- `cherry-off` is probably also off the table for lack of use cases.

**Comment by jguddas** [comment]
I like that aproach, do you feel like the use cases described below are sufficient?
If you can come up with a use case for `cherry-off` or any of the other icons let me know.

|name|icon|use case|
|---|---|---|
|album-off|![album-off](https://user-images.githubusercontent.com/25524993/214830748-1f5a422e-ebf2-4a82-95e5-f195020fd57e.svg)|remove bookmarks|
|alert-triangle-off|![alert-triangle-off](https://user-images.githubusercontent.com/25524993/214830746-9a81c412-1196-42ec-a004-62bb808c1c25.svg)|disable alerts|
|archive-off|![archive-off](https://user-images.githubusercontent.com/25524993/214830742-b2145a9f-9d85-46df-8f79-35b6c61c20d3.svg)|remove archive|
|armchair-off|![armchair-off](https://user-images.githubusercontent.com/25524993/214830740-ab109da9-3b9c-4829-8490-ee301879e22d.svg)|apartment is unfurnished|
|award-off|![award-off](https://user-images.githubusercontent.com/25524993/214830739-a23aace3-ab02-4941-828f-958ce7e874f3.svg)|leave leaderboard|
|beer-off|![beer-off](https://user-images.githubusercontent.com/25524993/214830738-e01a9c2d-3594-437b-a903-90bf7242fb3f.svg)|no beers / no alcohol|
|book-off|![book-off](https://user-images.githubusercontent.com/25524993/214830732-7c58d5fa-2015-43bb-826d-44d92b382003.svg)||
|bookmark-off|![bookmark-off](https://user-images.githubusercontent.com/25524993/214830734-69b6c513-8279-4bc5-a04c-5d266368d781.svg)|remove bookmark|
|briefcase-off|![briefcase-off](https://user-images.githubusercontent.com/25524993/214830730-28972108-8a39-4946-a48b-3ab4d2fa0475.svg)|luggage free checkin|
|bus-off|![bus-off](https://user-images.githubusercontent.com/25524993/214830727-ce7d9bfb-eaab-4653-b153-415a5da18e04.svg)|no bus allowed|
|car-off|![car-off](https://user-images.githubusercontent.com/25524993/214830723-86183d50-e262-409a-9489-477708a1240c.svg)|no cars allowed|
|cat-off|![cat-off](https://user-images.githubusercontent.com/25524993/214830721-ebc7c418-1e5c-4b82-a4d4-68104257dd14.svg)|no cats allowed|
|chef-hat-off|![chef-hat-off](https://user-images.githubusercontent.com/25524993/214830720-d17b7fee-15cf-43bc-bf61-df04e20fb7da.svg)|no cooking|
|cherry-off|![cherry-off](https://user-images.githubusercontent.com/25524993/214830719-d7b88e94-0869-4baf-b051-784a5c9aba7c.svg)||
|clapperboard-off|![clapperboard-off](https://user-images.githubusercontent.com/25524993/214830717-dc447b8c-2bb8-4901-ba81-05b7d019e0de.svg)|remove films|
|codepen-off|![codepen-off](https://user-images.githubusercontent.com/25524993/214830716-0fa0e8fc-1bc1-4061-a7c6-0323f7a0a579.svg)||
|concierge-bell-off|![concierge-bell-off](https://user-images.githubusercontent.com/25524993/214830715-650b8e13-b600-4e25-b7db-7904e57b5525.svg)|no service available|
|cross-off|![cross-off](https://user-images.githubusercontent.com/25524993/214830712-942e02c5-6776-4e9e-9682-99b79f567fdd.svg)||
|cup-soda-off|![cup-soda-off](https://user-images.githubusercontent.com/25524993/214830711-4cdf5811-f38e-4e50-bc18-58112216b0c0.svg)|no drinks available|
|curly-braces-off|![curly-braces-off](https://user-images.githubusercontent.com/25524993/214830710-9c7e48b8-bd57-4a7f-ba3d-e9417c7e8160.svg)|no code|
|database-off|![database-off](https://user-images.githubusercontent.com/25524993/214830706-86a4e0ed-b32b-4a91-bfef-17114bfc48ec.svg)|no database connected|
|diamond-off|![diamond-off](https://user-images.githubusercontent.com/25524993/214830705-4a394059-9ea2-4bbb-8447-f6bf2ee8d508.svg)||
|dog-off|![dog-off](https://user-images.githubusercontent.com/25524993/214830702-b72c0761-fbfc-4021-a781-f6fa85288fc7.svg)|no dogs / dogs forbiden|
|droplet-off|![droplet-off](https://user-images.githubusercontent.com/25524993/214830700-26aba846-199d-4123-9d5f-d2be0f1e4ad9.svg)|no watter supply / off-grid|
|facebook-off|![facebook-off](https://user-images.githubusercontent.com/25524993/214830698-76b3c93f-6206-46e5-8930-6fe84bccc6de.svg)|remove facebook intigration|
|fan-off|![fan-off](https://user-images.githubusercontent.com/25524993/214830696-123a6a86-deb6-4eaa-a829-ae6eb38df4f4.svg)|no air conditionining|
|feather-off|![feather-off](https://user-images.githubusercontent.com/25524993/214830694-e0864f97-135f-4bd3-9b71-5cf53029511c.svg)||
|file-off|![file-off](https://user-images.githubusercontent.com/25524993/214830693-7482c9b2-6bca-4182-a0c9-fbf1e94628a8.svg)||
|fingerprint-off|![fingerprint-off](https://user-images.githubusercontent.com/25524993/214830692-bfa98baf-453a-4f88-ad7a-0342dd52daef.svg)|remove fingerprint|
|flame-off|![flame-off](https://user-images.githubusercontent.com/25524993/214830690-41ef8e10-12a7-4828-81fc-e33c36df98b3.svg)|you have broken your streak|
|flower-2-off|![flower-2-off](https://user-images.githubusercontent.com/25524993/214830689-95d26d43-898c-4b77-b01d-94e6ad67dc4e.svg)||
|flower-off|![flower-off](https://user-images.githubusercontent.com/25524993/214830688-992e64b7-5829-46b7-a12f-83f7e4947518.svg)||
|folder-closed-off|![folder-closed-off](https://user-images.githubusercontent.com/25524993/214830687-b6b0b7f1-b327-4c04-b40d-02b7a0b914c5.svg)||
|folder-off|![folder-off](https://user-images.githubusercontent.com/25524993/214830685-d06432dd-7903-4308-8d9d-cdbd1fd81382.svg)||
|folder-open-off|![folder-open-off](https://user-images.githubusercontent.com/25524993/214830681-00857382-29de-4c1b-9f12-1f5774a6cf71.svg)||
|gift-off|![gift-off](https://user-images.githubusercontent.com/25524993/214830680-e9105f99-6f1a-4b1b-b29a-2b958c3f519a.svg)||
|glass-water-off|![glass-water-off](https://user-images.githubusercontent.com/25524993/214830679-79e6b699-c6a3-4daa-abad-3e18e405079e.svg)|no drinks available|
|hand-off|![hand-off](https://user-images.githubusercontent.com/25524993/214830678-2c6c58f3-ad26-4f3d-86b7-d3d0111a1984.svg)|forbid touching|
|hard-drive-off|![hard-drive-off](https://user-images.githubusercontent.com/25524993/214830675-93ec7421-bb00-4c2e-8ecc-63d070ceecad.svg)||
|hash-off|![hash-off](https://user-images.githubusercontent.com/25524993/214830673-28d043d8-4aa5-4efd-9938-b0a472dd3cf9.svg)||
|ice-cream-off|![ice-cream-off](https://user-images.githubusercontent.com/25524993/214830671-88e048c0-44ed-4349-a953-ff51ec9e5bcf.svg)||
|inbox-off|![inbox-off](https://user-images.githubusercontent.com/25524993/214830670-bcc59cf7-c7cc-4776-8164-9e26c47dd782.svg)|disable mail|
|leaf-off|![leaf-off](https://user-images.githubusercontent.com/25524993/214830667-73859a18-c965-41c5-981e-346df668d8a7.svg)||
|mail-open-off|![mail-open-off](https://user-images.githubusercontent.com/25524993/214830664-c5805513-cac9-472e-91b5-37650ef45a4a.svg)|disable mail|
|map-off|![map-off](https://user-images.githubusercontent.com/25524993/214830661-6f9a0f4e-5b6d-49e7-9d9c-0b4bb578ac39.svg)|disable map|
|medal-off|![medal-off](https://user-images.githubusercontent.com/25524993/214830658-6b2a9d28-9d6c-4b0a-a377-d42090692953.svg)|disable leaderboard|
|message-circle-off|![message-circle-off](https://user-images.githubusercontent.com/25524993/214830657-6b6a19f0-cfdc-4e5d-af16-e7ed40f0f323.svg)|disable chat|
|message-square-off|![message-square-off](https://user-images.githubusercontent.com/25524993/214830655-ec9e8b32-b943-4f8e-b05d-20da45da86bc.svg)|disable chat|
|palmtree-off|![palmtree-off](https://user-images.githubusercontent.com/25524993/214830654-a36097d4-679c-4f58-b0c8-4b34b0d1d7eb.svg)||
|paperclip-off|![paperclip-off](https://user-images.githubusercontent.com/25524993/214830652-cf950c7d-9807-414a-89a0-d673da7c0e92.svg)|no attachments|
|piggy-bank-off|![piggy-bank-off](https://user-images.githubusercontent.com/25524993/214830650-1a4eb9d7-28d0-4c6c-8f7f-0561b380d2e2.svg)||
|plug-off|![plug-off](https://user-images.githubusercontent.com/25524993/214830647-17d9d332-baeb-4db5-9d01-834c4f6d066a.svg)||
|podcast-off|![podcast-off](https://user-images.githubusercontent.com/25524993/214830645-f8252fd8-934f-42fb-bc6d-47e352d200b3.svg)||
|pointer-off|![pointer-off](https://user-images.githubusercontent.com/25524993/214830644-07eca6a8-838e-4b4b-9c32-5e0471f6251b.svg)||
|printer-off|![printer-off](https://user-images.githubusercontent.com/25524993/214830642-b2aa0e34-2d6c-48d3-b02d-846fe040b828.svg)|disable printer|
|quote-off|![quote-off](https://user-images.githubusercontent.com/25524993/214830641-104ae9a5-a8b9-4394-90cb-710e35c3d586.svg)|unquote|
|scale-off|![scale-off](https://user-images.githubusercontent.com/25524993/214830636-b909da28-9382-4f13-92cb-e2a271344ae2.svg)|disable rules|
|scroll-off|![scroll-off](https://user-images.githubusercontent.com/25524993/214830633-1d83b542-aa25-4481-8206-f3377b3274b1.svg)||
|settings-off|![settings-off](https://user-images.githubusercontent.com/25524993/214830632-adf6df26-288e-4dda-bb8e-0b074975b849.svg)|settings disabled|
|share-2-off|![share-2-off](https://user-images.githubusercontent.com/25524993/214830630-9f1cc57f-d044-47b4-a833-42f0a08d70e1.svg)|sharing disabled|
|shirt-off|![shirt-off](https://user-images.githubusercontent.com/25524993/214830629-f9833de1-9c74-4a39-9ee0-bf0a15ffab01.svg)|no shopping|
|siren-off|![siren-off](https://user-images.githubusercontent.com/25524993/214830628-19c1a6a3-b548-462b-8126-a70a1461f88a.svg)|alarm disabled|
|sofa-off|![sofa-off](https://user-images.githubusercontent.com/25524993/214830625-b88aba29-6c8e-4a0a-8e90-f45d3017c24d.svg)||
|sprout-off|![sprout-off](https://user-images.githubusercontent.com/25524993/214830623-eb3862b9-2c8a-401f-b44e-46ad096c5541.svg)|no plants available|
|sticky-note-off|![sticky-note-off](https://user-images.githubusercontent.com/25524993/214830619-cecde2c1-1733-4fb9-8132-ddcdc3975bed.svg)||
|target-off|![target-off](https://user-images.githubusercontent.com/25524993/214830618-e1e0f2b3-cd41-4bfe-8d41-e235c749af5f.svg)||
|text-cursor-off|![text-cursor-off](https://user-images.githubusercontent.com/25524993/214830617-ea4cb493-0223-4a06-b992-5d20fabbb03a.svg)|input disabled|
|thumbs-down-off|![thumbs-down-off](https://user-images.githubusercontent.com/25524993/214830615-cf705777-a784-4289-9407-efcbdee16a7a.svg)|disliking disabled|
|thumbs-up-off|![thumbs-up-off](https://user-images.githubusercontent.com/25524993/214830614-b1262c96-aee1-4d41-abda-21221a7b1c82.svg)|liking disabled|
|ticket-off|![ticket-off](https://user-images.githubusercontent.com/25524993/214830609-86cd6c24-f21a-445d-9dd7-3a5658aec7cb.svg)|no tickets available|
|trash-2-off|![trash-2-off](https://user-images.githubusercontent.com/25524993/214830606-11766df8-b377-46f1-b80d-accd8eb0bb97.svg)|deleting disabled|
|trash-off|![trash-off](https://user-images.githubusercontent.com/25524993/214830605-76b2b188-f116-4115-a096-b70ec55c7015.svg)|deleting disabled|
|tree-deciduous-off|![tree-deciduous-off](https://user-images.githubusercontent.com/25524993/214830602-d4b9ee0a-ed4f-4c93-bd83-99f98a49fcf5.svg)|no trees available|
|tree-pine-off|![tree-pine-off](https://user-images.githubusercontent.com/25524993/214830600-39cd3306-4bb8-4149-9502-bb0b6ae5b71a.svg)|no pine trees available|
|trello-off|![trello-off](https://user-images.githubusercontent.com/25524993/214830597-26814c24-a68f-4386-83d1-91c1d5204432.svg)||
|type-off|![type-off](https://user-images.githubusercontent.com/25524993/214830595-18cc2666-ab7b-4239-9548-c7cc023aaf13.svg)||
|umbrella-off|![umbrella-off](https://user-images.githubusercontent.com/25524993/214830592-2f4ee8f4-8252-4bdd-81af-da4ecb61003d.svg)|mark something as unprotected|
|unlock-off|![unlock-off](https://user-images.githubusercontent.com/25524993/214830590-3ac15739-ae80-4fcc-9d45-f388f81e635b.svg)||
|vegan-off|![vegan-off](https://user-images.githubusercontent.com/25524993/214830588-a1b48dd9-2f3a-4b25-8691-135fc6843ec1.svg)|mark something as not vegan|
|venetian-mask-off|![venetian-mask-off](https://user-images.githubusercontent.com/25524993/214830584-df95bbcb-b520-46e9-ad42-2bd93daab06f.svg)|leave incognito mode|
|view-off|![view-off](https://user-images.githubusercontent.com/25524993/214830582-8b26bc0c-f823-40d2-b1cb-7d8e84ab48fc.svg)||
|watch-off|![watch-off](https://user-images.githubusercontent.com/25524993/214830581-6c277886-b2f9-41cd-8b2e-7bd472c74e7e.svg)||
|webcam-off|![webcam-off](https://user-images.githubusercontent.com/25524993/214830580-9f0bdc58-68e0-4e54-8a3b-93fa157273ec.svg)|disable webcam|
|webhook-off|![webhook-off](https://user-images.githubusercontent.com/25524993/214830576-af1dcb64-4ce8-4dee-8883-5306ce5a58d9.svg)|disable webhook|

**Comment by tryoxiss** [comment]
@jguddas 
- hash off : don't display tags / tags not allowed
- plug off : do not plug in 
- type off : cannot add text here
- unlock off : this setting cant be unlocked due to another setting
- hard drive off : do drive found
- leaf off : eco mode off

Not sure about some of these but I can defintely see some of them being used

**Comment by KayleyHill** [comment]
Robot off (once the new robot has been finalized), to symbolize no AI allowed (which will get more important as AI becomes more frequent).

**Comment by ArneReimann** [comment]
Do you plan to add the listed off versions?

I would be so happy about the archive-off icon.

My Use Case: 
I have a toggle for archived items. 
To show archived items, I use "archive" I use Currently I have to use eye-off 
To hide archived items, I currently have to use eye-off.

**Comment by AnnaSasDev** [comment]
When looking at all the `-off` icons, there is one in Lucide currently not following the same format as all the other `-off` icons: https://lucide.dev/icons/screen-share-off
Though I believe this icon can also co-exist with a new `screen-share-off` as maybe `screen-share-blocked`?

**Comment by karsa-mistmere** [comment]
> When looking at all the `-off` icons, there is one in Lucide currently not following the same format as all the other `-off` icons: https://lucide.dev/icons/screen-share-off Though I believe this icon can also co-exist with a new `screen-share-off` as maybe `screen-share-blocked`?

I think the main issue is that 
- `screen-share` should be `monitor-arrow-up-right`
- `screen-share-off` should be `monitor-x`

**Comment by ericfennis** [comment]
I'm converting this into a discussion since it is unclear when this will be resolved. 
The discussion is good, but it lacks a defined scope.

If someone needs one of the `-off` icons check out other issues: https://github.com/lucide-icons/lucide/issues?q=is%3Aissue%20state%3Aopen%20off
Or create a new one.
